File: CoffeeBack/src/main/java/br/ufsm/csi/coffee/model/Produto.java
package br.ufsm.csi.coffee.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Produto {
    private int id;
    private String nome;
    private double preco;
    private boolean indcozinha;
    private boolean ativo;
}

File: CoffeeFront/src/app/principal/produto/produto.ts
export class Produto{
    id?: number;
    nome?: string;
    preco?: number;
    indcozinha?: boolean;
    ativo?: boolean;
    
    constructor(id?: number, nome?: string, preco?: number, indCozina?: boolean){
        this.id = id;
        this.nome = nome;
        this.indcozinha = indCozina;
        this.preco = preco;
        this.ativo = true;
    }
}